### PR TITLE
Remove thes `FTP` from the prefix of project environment variables

### PR DESCRIPTION
**Pull request by stephane-klein**
As mentioned [here](https://notes.sklein.xyz/Notes-%C3%A9ph%C3%A9m%C3%A8res/2024-06-07_1712), what do you think of this pull request that removes `FTP` from the prefix of the project's environment variables?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,16 +66,18 @@ To configure `restic-ftp-docker`, you can use
...
+| RESTIC_DOCKER_KEEP_DAILY           | `7`         | argument for `--keep-daily` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy)   |
+| RESTIC_DOCKER_KEEP_WEEKLY          | `4`         | argument for `--keep-weekly` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy)  |
+| RESTIC_DOCKER_KEEP_MONTHLY         | `12`        | argument for `--keep-monthly` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy) |
+
+Note: the `RESTIC_DOCKER_` prefix is used for `restic-ftp-docker` projet environment variables. When `_DOCKER_` is not present in the prefix then these are [environment variables supported in the restic project](https://restic.readthedocs.io/en/stable/040_backup.html#environment-variables).
```
Comment (stephane-klein) [inline README.md]: @Its-Alex what do you think about this note? Is it easy to understand?

Comment (Its-Alex) [reply README.md]: @stephane-klein I think you can merge that text with https://github.com/Its-Alex/restic-ftp-docker/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R65 For example:

```markdown
To configure `restic-ftp-docker`, you can utilize environment variables from `restic` and environment variables from `restic-ftp-docker`, as documented below.

Please note that the `RESTIC_DOCKER_` prefix is used for `restic-ftp-docker` project environment variables. When the `_DOCKER_` suffix is not present in the prefix, these are the environment variables supported in the restic project.

For information on environment variables supported in the restic project, refer to [this documentation](https://restic.readthedocs.io/en/stable/040_backup.html#environment-variables).
```

Comment (stephane-klein) [reply README.md]: @Its-Alex :+1: good idea.

Comment (stephane-klein) [reply README.md]: @Its-Alex Done.

**Pull request merged**